Add guidance for international applicants

peteryates opened the pull request:
This is a direct import of content provided to us. It looks like it needs a few tweaks before it's published.


## Page balance

The heading is _rather long_ and dominates the page. It could be more succinct.

![Screenshot from 2021-03-01 10-12-16](https://user-images.githubusercontent.com/128088/109483409-19f33c00-7a77-11eb-92b9-435aa5110c9f.png)

## Linking from within headers

This should be avoided, the collective definition of the EEA/Switzerland should probably be defined on the page.

![Screenshot from 2021-03-01 10-16-24](https://user-images.githubusercontent.com/128088/109483536-44dd9000-7a77-11eb-8a5e-166067bbd3f9.png)

## Remove connecting words from lists

Typically we avoid this and each list entry is a standalone statement that follows on from the sentence preceding the list. Here's an example excerpt (note the `, and`):

```markdown
  - have settled or pre-settled status under the EU Settlement Scheme,
    and
  - have been resident in the UK, EEA or Switzerland throughout the 3
    years before the start of your course, and
  - are an EEA/Swiss worker or a family member of one
```

peteryates committed:
Add guidance for international applicants

This is a direct import of content provided to us. It looks like it
needs a few tweaks before it's published.

peteryates committed:
Add a couple of hyperlinks missed by Pandoc's importer

Looks like these were missed because the opening hyperlink encapsulates
the entire second list entry 🤷‍♂️

peteryates committed:
Remove a rogue left square bracket

peteryates commented:
One way to make the title a bit less imposing could be to break it up slightly and use a [caption](https://design-system.service.gov.uk/styles/typography#headings-with-captions) for the academic year info.

![Screenshot from 2021-03-01 13-45-22](https://user-images.githubusercontent.com/128088/109505548-9d6f5600-7a94-11eb-8f11-8a9559374c49.png)

becky-johnes commented:
> One way to make the title a bit less imposing could be to break it up slightly and use a [caption](https://design-system.service.gov.uk/styles/typography#headings-with-captions) for the academic year info.
> 
> ![Screenshot from 2021-03-01 13-45-22](https://user-images.githubusercontent.com/128088/109505548-9d6f5600-7a94-11eb-8f11-8a9559374c49.png)

This looks good, thank you.

becky-johnes commented:
> This is a direct import of content provided to us. It looks like it needs a few tweaks before it's published.
> 
> ## Page balance
> The heading is _rather long_ and dominates the page. It could be more succinct.
> 
> ![Screenshot from 2021-03-01 10-12-16](https://user-images.githubusercontent.com/128088/109483409-19f33c00-7a77-11eb-92b9-435aa5110c9f.png)
> 
> ## Linking from within headers
> This should be avoided, the collective definition of the EEA/Switzerland should probably be defined on the page.
> 
> ![Screenshot from 2021-03-01 10-16-24](https://user-images.githubusercontent.com/128088/109483536-44dd9000-7a77-11eb-8a5e-166067bbd3f9.png)
> 
> ## Remove connecting words from lists
> Typically we avoid this and each list entry is a standalone statement that follows on from the sentence preceding the list. Here's an example excerpt (note the `, and`):
> 
> ```gfm
>   - have settled or pre-settled status under the EU Settlement Scheme,
>     and
>   - have been resident in the UK, EEA or Switzerland throughout the 3
>     years before the start of your course, and
>   - are an EEA/Swiss worker or a family member of one
> ```

On defining EEA/Switzerland - we provide a definition at the top of the section specifically for applicants from the EEA and Switzerland ('The EEA includes EU countries, plus Iceland, Liechtenstein and Norway.'). Maybe we could also include this link on the word 'EEA': https://www.gov.uk/eu-eea

This does mean that we use EEA in the first section before we've defined it - we could add a definition in here but it's probably a bit repetitive / distracts from the main point.

becky-johnes commented:
> This is a direct import of content provided to us. It looks like it needs a few tweaks before it's published.
> 
> ## Page balance
> The heading is _rather long_ and dominates the page. It could be more succinct.
> 
> ![Screenshot from 2021-03-01 10-12-16](https://user-images.githubusercontent.com/128088/109483409-19f33c00-7a77-11eb-92b9-435aa5110c9f.png)
> 
> ## Linking from within headers
> This should be avoided, the collective definition of the EEA/Switzerland should probably be defined on the page.
> 
> ![Screenshot from 2021-03-01 10-16-24](https://user-images.githubusercontent.com/128088/109483536-44dd9000-7a77-11eb-8a5e-166067bbd3f9.png)
> 
> ## Remove connecting words from lists
> Typically we avoid this and each list entry is a standalone statement that follows on from the sentence preceding the list. Here's an example excerpt (note the `, and`):
> 
> ```gfm
>   - have settled or pre-settled status under the EU Settlement Scheme,
>     and
>   - have been resident in the UK, EEA or Switzerland throughout the 3
>     years before the start of your course, and
>   - are an EEA/Swiss worker or a family member of one
> ```

It's really important that we include the 'and' each time it's used in these lists. We need to be really clear that someone must meet all the criteria outlined in the bullet point lists - it's not enough for someone to meet just one or two and it would be really misleading if we were to imply that this is the case (would make more people think they are eligible for significant financial support when they actually are not). Please can we keep this as it is?

becky-johnes commented:
In the 'Check what support is available' section, there is a full stop after the second bullet point. I think this should be removed so that it's in line with the formatting used on the rest of the page? (Sorry - probably something that's been pulled through from our drafting.)

Also, on the 'How to apply' section - please can the link that is on the word 'bursary' be linked to 'bursary or scholarship' - as the information applies to both?

becky-johnes commented:
It looks as though this funding guidance will be on a standalone page - is that right? I think that works fine but would just want to make sure that we link to it from the main page for international applicants: https://beta-getintoteaching.education.gov.uk/international-candidates/ - this has a short section on 'Funding' within the 'How to apply to train to teach' section. I think that section will need a bit of re-drafting so that the link to this guidance makes sense. Shall I send over some wording?

The main funding page has a section called 'Applying for funding if you come from outside England' - https://beta-getintoteaching.education.gov.uk/international-candidates/ - maybe it would make sense to include a link to this page on funding for international applicants there too?

peteryates committed:
Remove unnecessary full stop from list

Responding to requests made by @becky-johnes

peteryates committed:
Include 'or scholarship' in training provider link

Responding to requests made by @becky-johnes

peteryates committed:
Remove the env-specific prefix on internal links

Also target the right part of the page via anchors

peteryates committed:
Move the academic year from title to a caption

This will display the year in a slightly-lighter grey above the title,
allowing us to shorten it slightly.

peteryates committed:
Add a link to the EEA definition list and clarify

Replacing 'these countries' with 'the EEA' makes it clear exactly
who we're referring to, it also gives us the opportunity to insert a
hyperlink.

Refs #329

peteryates committed:
Add link to int'l applicants from funding page

peteryates commented:
@becky-johnes - I've made the following changes based on your recommendations:

* removed the unnecessary full stop (a156ade9f9e)
* included 'or scholarship' in the bursary link text (5aa79ceb)
* moved the academic year information to a caption above the main title (4a4a9dfdb)
* slightly reworded the "if you're an applicant from these countries" sentence in the _Applicants from outside the EEA_ section, it now says:
 
  > If you are an applicant from outside [the EEA](https://www.gov.uk/eu-eea) or Switzerland, you are unlikely to be eligible for financial support.

  I think this is clear and puts the link to the full definition right at the top. (93eb69306d4)
* added a link to new content from the "Funding your training" page (07a52c56b69)


> I think that section will need a bit of re-drafting so that the link to this guidance makes sense. *Shall I send over some wording?*

Please do and I'll make the necessary changes.

becky-johnes commented:
Thanks very much Peter. I'll email you proposed wording for the main funding page and the main international applicants page.

peteryates committed:
Standardise apostrophes

peteryates committed:
Add provided funding paragraph to int'l candidates

This is a draft, there are some notes beneath it on things we'll
probably need to revisit.

peteryates committed:
Attempt to international clarify funding options

peteryates committed:
Simplify the funding section even further

This content was provided by @laurahermionetennant92 - thanks for
shining the clarity torch on it! 😁

peteryates committed:
Clarify funding info for people outside England

peteryates committed:
Make funding page content match int'l candidates

peteryates committed:
Add acronym definition for the EEA

MylesJarvis reviewed (dismissed):
LGTM

peteryates committed:
Add table of contents to int'l teacher recruitment
